data/src/main/java/me/androidbox/data/remote/model/request/EventRequestModel.kt

```diff
@@ -0,0 +1,11 @@
+package me.androidbox.data.remote.model.request
+
+data class EventRequestModel(
+    val id: String,
+    val title: String,
+    val description: String,
+    val from: Long,
+    val to: Long,
+    val remindAt: Long,
+    val listOfAttendee: List<String>
```

Comment:
Personal opinion: Naming this just `attendees` says it all while being shorter. Same for your other list namings.
Also, if I remember correctly, an attendee is more than just a string here

Reply:
@philipplackner 
The naming convention I think I read from some coding standard. However, I would like to stay with it for consistency.